docs/src/increase-observability/metrics/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Monitor metrics
+weight: 5
+description: See all of the live infrastructure metrics available to give you an overview of resource usage.
+aliases:
+  - /dedicated/architecture/metrics.html
+---
+
+Platform.sh projects are accompanied by live infrastructure metrics that provide an overview of resource usage for environments.  
+
+Within the console, metrics can be found for an environment under **Metrics**:
+
+![A screenshot of what the metrics dashboard displays](/images/metrics/all.png "0.65")
+
+The information under **Metrics** shows [usage metrics](#examples-of-how-to-read-metrics) for:
+
+* [Dedicated environments](../../dedicated/overview/_index.md):
+  each of the three hosts in your [N+1 configuration](../../dedicated/architecture/_index.md) and their average
```

Comment:
under Dedicated show the Dedicated graph, which is the one that was above

![A screenshot of what the metrics dashboard displays](/images/metrics/all.png "0.65")